test/support/test_config.h

```diff
@@ -230,4 +230,7 @@
 #define _PSTL_RED_BY_SEG_WINDOWS_COMPILE_ORDER_BROKEN                                                                  \
     (_MSC_VER && TEST_DPCPP_BACKEND_PRESENT && __INTEL_LLVM_COMPILER <= 20250000)
 
+//oneAPI DPC+/C+ compiler produces 'Unexpected kernel lambda size issue' error
+#define _PSTL_LAMBDA_PTR_TO_MEMBER_WINDOWS_BROKEN (_MSC_VER && TEST_DPCPP_BACKEND_PRESENT && __INTEL_LLVM_COMPILER)

```

Comment:
Let's limit the broken macro with a specific release not to forget to remove it once the fix is available. I'd suggest a 2025.1 release.

Reply:
The broken macros expires after 2025.1 release:

```suggestion
#define _PSTL_LAMBDA_PTR_TO_MEMBER_WINDOWS_BROKEN (_MSC_VER && TEST_DPCPP_BACKEND_PRESENT && __INTEL_LLVM_COMPILER <= 20250100)

```